uint32 id
string type
std_msgs/ColorRGBA color
uint64 x_dim
uint64 y_dim
uint64 z_dim
geometry_msgs/Pose pose
